### ISSUE TITLE
dorotheum.com not supported

### ISSUE THREAD

**Issue by ilpanik** [lovasoa/dezoomify#750]
### Site name and desciption
What is the name and the URL of the site you would like dezoomify to support ?
https://www.dorotheum.com/it/l/7353562/

### Current error message
Describe the error dezoomify currently gives you when you try to dezoomify images from this site.

Uncaught Error: Unable to find a proper dezoomer for:
https://www.dorotheum.com/it/l/7353562/

(https://dezoomify.ophir.dev/dezoomers/automatic.js:30)

I looked to see if there was an issue already reported and found this (https://github.com/lovasoa/dezoomify/issues/532#issue-817821904) , but unfortunately I still wasn't able to resolve the problem.
I tried to follow the various guides you made, but probably due to my incompetence I wasn't able to find a solution

**Comment by natrius** [comment]
This site "A Lounge Sofa" (at least currently) does not seem to have a zoomable picture. Right click "Safe image as" and safe the image. I think this could be closed.

**Comment by nick-s-b** [comment]
Many do have a zoomable photo. For example: https://www.dorotheum.com/en/l/9328327/

**Comment by natrius** [comment]
Yeah, i know. Still, this issue can be closed, because the link mentioned above does not have a zoomable picture  :) 

For the zoomable-ones: i explained how i did it here https://github.com/lovasoa/dezoomify/issues/839#issuecomment-2670874690

**Comment by AlexanderDeLargeBrotherSir** [comment]
@ilpanik, close this right now
